### DISCUSSION TITLE
Chiffres 2024

### DISCUSSION THREAD

**Discussion by Niicis** [betagouv/mon-entreprise#2860] [Q&A] [answered]
Bonjour, 

Quand seront disponible les chiffres et taux pour l'année 2024 ?

Merci d'avance,
Cordialement,

**Comment by johangirod** [comment, marked as answer]
Bonjour, 
Les simulateurs, l'API et le paquet NPM sont maintenant à jour avec les taux et barèmes de 2024.

Bien à  vous,

**Reply by Niicis** [reply]
Bonjour,

Merci pour votre retour, est ce que les caisses de retraite des professions libérales peuvent etre mises a jour svp ?

Cordialement,

**Reply by johangirod** [reply]
Les caisses de retraite CNAVPL mettent un peu de temps à publier les taux et barème de la nouvelle année à chaque fois. Nous les mettons à jour fin janvier / début février, afin de tout faire en même temps. Nous acceptons les PR entretemps. 

https://github.com/betagouv/mon-entreprise/issues/2864
